samples/bluetooth/throughput/src/main.c

```diff
@@ -29,8 +29,8 @@
 
 #define DEVICE_NAME	CONFIG_BT_DEVICE_NAME
 #define DEVICE_NAME_LEN (sizeof(DEVICE_NAME) - 1)
-#define INTERVAL_MIN	0x140	/* 320 units, 400 ms */
-#define INTERVAL_MAX	0x140	/* 320 units, 400 ms */
+#define INTERVAL_MIN	0x6	/* 6 units, 7.5 ms, only used to setup connection */
+#define INTERVAL_MAX	0x6	/* 6 units, 7.5 ms, only used to setup connection */
```

Comment:
is this to get the data length update instant faster?

Reply:
yes, and encrypting the link faster on initial connection.